.github/workflows/kotlin-sync.yml

```diff
@@ -75,4 +76,6 @@ jobs:
           }
           await github.issues.create({...context.repo, 
             title: 'Sync with Kotlin Compiler: ' + readFile("file:///home/runner/work/arrow-meta/arrow-meta/version"), 
-            body: 'Failure: \n' + readFile("file:///home/runner/work/arrow-meta/arrow-meta/stderr")});
+            body: ' * FAILURE: \n' + readFile("file:///home/runner/work/arrow-meta/arrow-meta/stderr"),
+            assignees: ['raulraja', 'i-walker', 'ahinchman1', 'rachelcarmena'],
```

Comment:
Can we use git blame in the file to populate this array?

Reply:
Good idea! I'm going to create an issue to test it with different types of errors (compilation or tests).